public/scripts/app.js

```diff
@@ -2,6 +2,94 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+// Profile GET Statement
+$.ajax({
+  method: "GET",
+  url:"/api/profile"
+})
+// DB GET Statement
+$.ajax({
+  method: "GET",
+  url: "/api/heroes",
+  success: displayHero
+})
 
+
+// Function to show seeded DB
+function displayHero(heroes){
```

Comment:
This function is named displayHero singular, but its parameter is heroes plural; this is one of those things that works, but will trip you up if you ever forget that this function is named singularly but takes plural arguments.